### DISCUSSION TITLE
阅读模式下代码高亮问题

### DISCUSSION THREAD

**Discussion by dongcj** [tamlok/vnote#1857] [Q&A] [answered]
在 编辑模式 中，代码高亮很好，但是在 阅读模式 中，很多代码显示不出来，比如用 ```sh 的这种代码。
请问如何在 阅读模式 中，显示出高亮？

**Comment by tamlok** [comment, marked as answer]
Hi,

具体语言对应的缩写请参考官网：https://prismjs.com/#supported-languages
